charts/argo-services/templates/argo-workflows-rbac/role.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
can you add a reference as a comment if you got the resources list from somewhere?